EpikCord/__init__.py

```diff
@@ -1156,6 +1113,154 @@ async def command_error(
     ):
         logger.exception(error)
 
+    #AutoModeration Events
+    async def auto_moderation_rule_create(self, data:dict) -> AutoModerationRule:
+        rule = AutoModerationRule(self, data)
+        self.auto_moderation_cache.add_to_cache(rule.guild_id, rule)
```

Comment:
Again, should be an attribute of Guild. Not the Client.

Reply:
I do not understand